packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -204,22 +213,26 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
 
   public async launchCommunityFromStorage() {
     this.logger.info('Launching community from storage')
-
     const community = await this.localDbService.getCurrentCommunity()
-    // TODO: Revisit this when we move the Identity model to the backend, since
-    // this network data lives in that model.
-    const network = await this.localDbService.getNetworkInfo()
-
-    if (community && network) {
-      const sortedPeers = await this.localDbService.getSortedPeers(community.peerList ?? [])
-      this.logger.info('launchCommunityFromStorage - sorted peers', sortedPeers)
-      if (sortedPeers.length > 0) {
-        community.peerList = sortedPeers
-      }
-      await this.localDbService.setCommunity(community)
+    if (!community) {
+      this.logger.info('No community found in storage')
+      return
+    }
+
+    const identity = await this.storageService.getIdentity(community.id)
+    if (!identity) {
+      this.logger.info('No identity found in storage')
```

Comment:
I made the identity missing a warning, but left the community missing an info because the community missing is normal when initializing the app prior to joining a community. Not having an identity while having a community is not normal though.